libs/ledger-live-common/src/families/icon/setup.ts

```diff
@@ -0,0 +1,23 @@
+// Goal of this file is to inject all necessary device/signer dependency to coin-modules
+
+import { createBridges } from "@ledgerhq/coin-icon/bridge/js";
```

Comment:
```suggestion
import { createBridges } from "@ledgerhq/coin-icon/bridge/index";
```

it's index.ts now right ?

Reply:
Sorry, My bad. I fixed it. thank you @lvndry